book/01-introduction/sections/02-installing.asc

```diff
@@ -21,6 +21,7 @@ Here you can download the `atom-mac.zip` file explicitly.
 
 Once you have that file, you can click on it to extract the binary and then drag the new `Atom` application into your ``Applications'' folder.
 
+We also recommend you run `Atom->Install Shell Commands` from the Atom menu so you can run the `atom` and `apm` from terminal.
```

Comment:
We also recommend that you install the shell commands (`Install Shell Commands` from the Command Palette`) so that you can run the `atom` and `apm` commands from the terminal.

How does that sound to you?